pkg/client/helper.go

```diff
@@ -245,8 +246,9 @@ func InClusterConfig() (*Config, error) {
 		Host:        "https://" + net.JoinHostPort(os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")),
 		Version:     "v1beta3",
 		BearerToken: string(token),
-		// TODO: package certs along with the token
-		Insecure: true,
+		TLSClientConfig: TLSClientConfig{
```

Comment:
@liggitt suggested allowing an attempt to see if the cert is accepted using the default CA roots